### ISSUE TITLE
TemplateDoesNotExist on base.html extending (django 2.2.3) .

### ISSUE THREAD

**Issue by VitorSDDF** [drager/django-simple-blog#25]
I get the TemplateDoesNotExist error in those templates that extend base.html. I've seen that the project doesn't have a base.html template.I have to write mine?I'm using django 2.2.3 and python 3.5.2 .Thank you!

**Comment by drager** [comment]
Yeah, if you want to use the provided templates you need to add a `base.html` template. This is so you can customize the looks as you want.
